lambdas/indexer/openapi.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This looks like it wasn't updated in the subsequent commit. Furthermore, the build is green so that means something is broken with the freshness check on this file during the GitHub build.

Reply:
https://github.com/DataBiosphere/azul/issues/5228